sdk/deviceupdate/Azure.IoT.DeviceUpdate/src/autorest.md

```diff
@@ -9,9 +9,46 @@ Run `generate.ps1` or `dotnet msbuild /t:GenerateCode` to generate code.
 public-clients: true
 title: DeviceUpdate
 
-input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/b9b91929c304f8fb44002267b6c98d9fb9dde014/specification/deviceupdate/data-plane/Microsoft.DeviceUpdate/preview/2022-07-01-preview/deviceupdate.json
+input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/7c840caa77ac200f43636930d82fc31cf117241e/specification/deviceupdate/data-plane/Microsoft.DeviceUpdate/stable/2022-10-01/deviceupdate.json
 
 namespace: Azure.IoT.DeviceUpdate
 security: AADToken
 security-scopes:  https://api.adu.microsoft.com/.default
 ```
+
+### Fix 304s
+``` yaml
+directive:
+- from: swagger-document
+  where: $["paths"]["/deviceUpdate/{instanceId}/updates/providers/{provider}/names/{name}/versions/{version}"]
+  transform: >
+    $.get.responses["304"] = {
+      "description": "The condition specified using HTTP conditional header(s) is not met.",
+      "x-az-response-name": "ConditionNotMetError",
+      "headers": { "x-ms-error-code": { "x-ms-client-name": "ErrorCode", "type": "string" } }
```

Comment:
Does your service return an `x-ms-error-code` header on 304?  If not, you can delete these lines too.  They won't affect code gen for a protocol method, but may surface the value in a high level convenience in the future.